package.json

```diff
@@ -61,9 +65,7 @@
     "mocha": "^2.2.5",
     "phantomjs": "^1.9.17",
     "react-hot-loader": "^1.2.8",
-    "react-router": "^0.13.3",
-    "react-stub-context": "^0.3.0",
```

Comment:
Does this means that we can't fix the build until `react-stub-context` support react v0.14? 
I will close https://github.com/callemall/material-ui/pull/1725 when the build pass since I believe this PR is more appropriate.